OpenRA.Mods.Common/Widgets/SupportPowerTimerWidget.cs

```diff
@@ -19,47 +19,165 @@ namespace OpenRA.Mods.Common.Widgets
 {
 	public class SupportPowerTimerWidget : Widget
 	{
+		public readonly int XObserver = 10;
```

Comment:
Should I move the yaml definitions from ingame.yaml to ingame-player.yaml and ingame-observer.yaml? They each could have different X and Y values. But then there would be two separate definitions for SupportPowerTimer@SUPPORT_POWER_TIMER: